apps/user_profile/models.py

```diff
@@ -37,6 +39,34 @@ class UserProfile(models.Model):
         verbose_name='Номер телефону'
     )
 
+    city = models.ForeignKey(
+        to=City,
+        default=1
```

Comment:
don't use default value for FK, never

Reply:
when i try to migrate without default value any of fields, it shows 
(mvenv) riaboshtan@riaboshtan-E502NA:~/final_project$ python manage.py makemigrations
You are trying to add a non-nullable field 'city' to userprofile without a default; we can't do that (the database needs something to populate existing rows).
Please select a fix:
 1) Provide a one-off default now (will be set on all existing rows with a null value for this column)
 2) Quit, and let me add a default in models.py
Select an option: